azure-pipelines.yml

```diff
@@ -78,26 +78,26 @@ jobs:
         displayName: Install Az Modules
         inputs:
           targetType: 'inline'
-          script: 'Install-Module Az.Accounts, Az.KeyVault, Az.Resources -Force'
+          script: 'Install-Module Az.Accounts, Az.KeyVault, Az.Resources, Posh-ACME -Force'
           errorActionPreference: 'stop'
           failOnStderr: true
           pwsh: true
 
-      # 3. Install Posh-ACME PowerShell module from a local NuGet package
-      - task: PowerShell@2
-        displayName: Install Posh-ACME Module
-        inputs:
-          targetType: 'inline'
-          script: |
-            Expand-Archive -LiteralPath (Resolve-Path "./Packages/*.nupkg" | Select -ExpandProperty Path) -DestinationPath ./pwsh_modules/Posh-ACME
-            $env:PSModulePath += ":./pwsh_modules"
-            echo "##vso[task.setvariable variable=PSModulePath]$env:PSModulePath"
-            Get-Module -ListAvailable -Verbose Posh-ACME
-          errorActionPreference: 'stop'
-          failOnStderr: true
-          pwsh: true
+      # # 3. Install Posh-ACME PowerShell module from a local NuGet package
```

Comment:
@raicastino why did you install the module Posh-ACME from the file system instead of using the Install-Module command?

Reply:
@uolter For security reasons described in this internal document: 
https://docs.google.com/document/d/1l4fC2giivCyPA2rwqrTwpue9c5ZKxaXw59l_SY4GFE4/edit#heading=h.rv3a08tsq4l5